astropy/utils/masked/function_helpers.py

```diff
@@ -138,7 +138,7 @@
     # Safe in < 2.0, because it deferred to the method. Overridden in >= 2.0.
     MASKED_SAFE_FUNCTIONS |= {np.ptp}
     # Removed in numpy 2.0.  Just an alias to vstack.
-    MASKED_SAFE_FUNCTIONS |= {np.row_stack}
+    MASKED_SAFE_FUNCTIONS |= {np.vstack}
```

Comment:
I don't think this change should happen, right?

Reply:
No, this is ruff trying to fix our code for v2, but we don't want that in this specific case.